server.go

```diff
@@ -8,20 +8,26 @@ import (
 	"net/http"
 	"bytes"
 	"strings"
-
-	"github.com/clarifai/clarifai-go"
 	"github.com/gin-gonic/gin"
+	"os"
+	"net/url"
 )
 
 const (
 	clarifaiApi = "https://api.clarifai.com/v2/models/aaa03c23b3724a16a56b629203edc62c/outputs"
+	clarifyAuth = "https://api.clarifai.com/v1/token"
```

Comment:
That's wired. As I saw they use v2 to get access_token in the guide example
 `curl -X POST \
    -u "$CLARIFAI_APP_ID:$CLARIFAI_APP_SECRET" \
    -d "grant_type=client_credentials" \
    https://api.clarifai.com/v2/token`
And all the other api endpoints use v2